pkg/charts/charts.go

```diff
@@ -24,8 +24,8 @@ type Chart struct {
 	Files     []*chartutil.BufferedFile
 }
 
-func (chart *Chart) render(partialsFiles []*chartutil.BufferedFile) (bytes.Buffer, error) {
-	if err := FilesReader(chart.Dir+"/", chart.Files); err != nil {
+func (c *Chart) render(partialsFiles []*chartutil.BufferedFile) (bytes.Buffer, error) {
+	if err := FilesReader(c.Dir+"/", c.Files); err != nil {
```

Comment:
A significant portion of this repository uses the single character notation, including files relatively close to this one.

Using that notation for this makes it possible to make the change below.